src/components/Login/Login.js

```diff
@@ -48,6 +51,8 @@ class Login extends Component {
       } else {
         Actions.generalInfo({ me: profile });
       }
+    } else {
```

Comment:
Now, it is possible situation which is that profile and error are all null.
So, I added else statement.